src/types/input_device.zig

```diff
@@ -49,4 +49,9 @@ pub const InputDevice = extern struct {
 
     extern fn wlr_input_device_get_virtual_keyboard(wlr_dev: *InputDevice) ?*wlr.VirtualKeyboardV1;
     pub const getVirtualKeyboard = wlr_input_device_get_virtual_keyboard;
+
+    extern fn wlr_libinput_get_device_handle(wlr_dev: *InputDevice) ?*LibinputDevice;
```

Comment:
This can't return null given the current wlroots implementation.
```suggestion
    extern fn wlr_libinput_get_device_handle(wlr_dev: *InputDevice) *LibinputDevice;
```